IFC4x3/Templates/Product Shape/Product Geometric Representation/Surface Geometry/Surface Sectioned Geometry/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Surface Sectioned Geometry" UniqueId="c62dfbec-49c6-41e3-836e-d494170ec2c8" Type="IfcElement">
+	<Rules>
+		<DocModelRuleAttribute Name="Representation">
+			<Rules>
+				<DocModelRuleEntity Name="IfcProductDefinitionShape">
+					<Rules>
+						<DocModelRuleAttribute Name="Representations">
+							<Rules>
+								<DocModelRuleEntity Name="IfcShapeRepresentation">
+									<Rules>
+										<DocModelRuleAttribute Name="ContextOfItems">
+											<Rules>
+												<DocModelRuleEntity Name="IfcGeometricRepresentationContext" />
+											</Rules>
+										</DocModelRuleAttribute>
+										<DocModelRuleAttribute CardinalityMin="1" CardinalityMax="1" Name="RepresentationIdentifier" Identification="Identifier">
+											<Rules>
+												<DocModelRuleEntity Name="IfcLabel">
+													<Rules>
+														<DocModelRuleAttribute Name="Value=Surface" Description="Value=Surface" />
+													</Rules>
+												</DocModelRuleEntity>
```

Comment:
This is the wrong encoding it should look the same as the RepresentationType Below, this needs correcting

Reply:
Seems like there is an error in the encoding at the level "Surface Geometry", right? Didn't touch that but will look into it.

Reply:
yes if you merge the develop you should get the correct reference on Surface Geometry that just needs adding like in sectionedsolidHorizontal

Reply:
```suggestion
												<DocModelRuleEntity Name="IfcLabel">
													<Rules>
														<DocModelRuleEntity xsi:nil="true" href="IfcLabel_212pjtt2TCSe6IuxauYKd"/>
													</Rules>
												</DocModelRuleEntity>
```